### PR TITLE
Android example version compatibility

### PR DESCRIPTION
**Pull request by charlesbvll**
## What does this implement/fix? Explain your changes.

Makes this example compatible with the versions `>=1.0.0` of `flwr`.

The code of the android app needs an update also, as currently there is the following issue: https://github.com/adap/flower/issues/1388

### CODE REVIEW

**Comment by charlesbvll** [comment]
The proto file update is a possible fix but testing still needs to be done.

File: examples/android/client/app/src/main/java/flwr/android_client/MainActivity.java [hunk 1]
```diff
@@ -127,61 +140,44 @@ public void connect(View view) {
...
 
-    private static class FlowerServiceRunnable implements GrpcRunnable {
-        private Throwable failed;
+    private static class FlowerServiceRunnable{
+        protected Throwable failed;
         private StreamObserver<ClientMessage> requestObserver;
-        @Override
-        public void run(FlowerServiceBlockingStub blockingStub, FlowerServiceStub asyncStub, MainActivity activity)
-                throws Exception {
+//        @Override
```
Comment (danieljanes) [inline examples/android/client/app/src/main/java/flwr/android_client/MainActivity.java:174]: Remove this commented-our line?

File: examples/android/client/app/src/main/java/flwr/android_client/MainActivity.java [hunk 2]
```diff
@@ -127,61 +140,44 @@ public void connect(View view) {
...
-                return "Failed to connect to the FL server \n" + sw;
-            }
-        }
-
-        @Override
-        protected void onPostExecute(String result) {
-            MainActivity activity = activityReference;
-            if (activity == null) {
-                return;
+    public void runGRPC(View view){
```
Comment (danieljanes) [inline examples/android/client/app/src/main/java/flwr/android_client/MainActivity.java:143]: We usually put just the first letter in caps (as in `GrpcRunnable`):
```suggestion
    public void runGrpc(View view){
```

**Review by danieljanes — CHANGES_REQUESTED** [review]
Looks good overall, just two small comments!

**Review by danieljanes — APPROVED** [review]
Lgtm, great to have this updated to 1.0+!

**Pull request merged**